lib/evm_logic.rb

```diff
@@ -269,7 +278,7 @@ def to_csv
         csv_max_date = [@ev.keys.max, @ac.keys.max, @pv.keys.max].max
         evm_date_range = (csv_min_date..csv_max_date).to_a
         # title
-        csv << ["DATE",evm_date_range].flatten!
+        csv << ['DATE', evm_date_range].flatten!
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.